tcf_website/management/commands/other_schools_department_fix.py

```diff
@@ -0,0 +1,117 @@
+    # Optional flag --verbose can be set to show output of the script
+    # Must be run in a separate terminal after already running docker-compose up
+    # May have to restart the docker container for database changes to be visible on the site
+    # Visually check that CS department has been moved to E School and that
+    # other schools have been split up.
+
+    help = 'Promotes subdepartments in \'Other Schools at the University of Virginia\' to departments'
+
+    def add_arguments(self, parser):
+        # Named (optional) arguments
+        parser.add_argument(
+            '--verbose',
+            action='store_true',
+            help='Verbose output',
+        )
+
+    def handle(self, *args, **options):
+
+        self.verbose = options['verbose']
+
+        # Take Other Schools at UVA and promote all subdepartments into departments
+        # Excluded schools not to split up
+        excluded = [
+            "College of Arts & Sciences",
+            "School of Engineering & Applied Science",
+            "UNKNOWN"]
+
+        # Check if this script has already been run
+        if School.objects.get(
+                name='Curry School of Education').department_set.count() > 1:
+            print('Already split! Aborting script...')
+            return
+
+        if self.verbose:
+            print('Splitting all other schools...')
+            dash = '-' * 155
+            print(dash)
+            print(
+                '{:<50s}{:<50s}{:<50s}'.format(
+                    'Subdepartment',
+                    'Department',
+                    'School'))
+            print(dash)
+
+        # Go through all schools that are not in the list of excluded
+        for school in School.objects.exclude(name__in=excluded):
+            # Go through all departments
+            for department in school.department_set.all():
+                # Go through all subdepartments
+                for subdepartment in department.subdepartment_set.all():
+                    if self.verbose:
+                        print(
+                            '{:<50s}{:<50s}{:<50s}'.format(
+                                subdepartment.name,
+                                department.name,
+                                school.name))
+                        print(dash)
+                    # Promote the subdepartment to a department
+                    new_department = Department(
+                        name=subdepartment.name,
+                        school=school,
+                        description=subdepartment.description
+                    )
+                    # Reassign relation
+                    subdepartment.department = new_department
+                    # Save everything
+                    new_department.save()
+                    subdepartment.save()
+                # Delete old department
+                department.delete()
+
+        # Moving Computer Science from A&S to School of Engineering
```

Comment:
feel like it might be better to do this in the scripts that migrate the legacy db (migrate_legacy_subdepartments?) when we do the final db transfer so it'll be fixed from the get-go